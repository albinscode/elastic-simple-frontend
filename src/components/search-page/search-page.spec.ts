import SearchPage from "./search-page.vue";
import Vue from "vue";

describe("Search page default tests", () => {
    it("renders correctly", done => {
        const comp = new Vue(SearchPage).$mount();
        comp.$nextTick(() => {
            expect(comp.$el).toMatchSnapshot();
            done();
        });
    });
});
