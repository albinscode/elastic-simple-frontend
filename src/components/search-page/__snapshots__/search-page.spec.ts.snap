// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search page default tests renders correctly 1`] = `
<div>
  <div
    class="form"
  >
    <input
      name="query"
      type="text"
    />
     
    <button>
      Rechercher
    </button>
  </div>
   
  <div
    class="results"
  >
     
    <!---->
  </div>
</div>
`;
